collaberation.txt

```diff
@@ -2,4 +2,5 @@ user b done these changes and pushed data
 This is to show how amend works
 adding one more line in main branch
 about merging
-data from main branch
\ No newline at end of file
+data from feature branch
```

Comment:
Why this change happen? Earlier base branch content was the first line but now it got change?
Any specific reasons for that?

Reply:
modified while merging or pulling from base branch